### DISCUSSION TITLE
Is there a way to use hardware RX/TX pins instead of the USB?

### DISCUSSION THREAD

**Discussion by cuishuang413** [grblHAL/core#233] [General]
Hello,

Is there a way to use hardware RX/TX pins instead of the USB? I'm not using any other serial devices and would like to control the grblHAL using the RX/TX pins on the board.

Thanks and happy holidays.

**Comment by terjeio** [comment]
Yes, most boards supports that. Comment out `USB_SERIAL_CDC` in _my_machine.h_ to enable UART comms.

**Reply by cuishuang413** [reply]
Oh.!! Okay. Thanks! Why didn't I try that already is what I'm wondering.

Thanks!
